xwiki-rendering-syntaxes/xwiki-rendering-syntax-xhtml/src/main/java/org/xwiki/rendering/internal/parser/xhtml/wikimodel/XWikiCommentHandler.java

```diff
@@ -127,8 +127,8 @@ private void handleLinkCommentStart(String content, TagStack stack)
 
         stack.pushStackParameter(LINK_LISTENER, xwikiListener);
 
-        stack.pushStackParameter(IS_IN_LINK, true);
-        stack.pushStackParameter(IS_FREE_STANDING_LINK, false);
+        stack.pushStackParameter(IS_IN_LINK, Boolean.TRUE);
+        stack.pushStackParameter(IS_FREE_STANDING_LINK, Boolean.FALSE);
```

Comment:
I'm curious to understand why this is better? I find it less good than what we had.

Reply:
http://sonar.xwiki.org/issues/search#issues=02a0f965-09f1-477f-ab62-6362ab587043